modules/hosted-control-planes-concepts-personas.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * architecture/control-plane.adoc
+// * hosted-control-planes/index.adoc
+
+
+:_content-type: CONCEPT
+[id="hosted-control-planes-concepts-personas_{context}"]
+= Glossary of common concepts and personas for hosted control planes
+
+When you use hosted control planes for {product-title}, it is important to understand its key concepts and the personas that are involved.
+
+Management cluster:: An {product-title} cluster that hosts hosted control planes and zero to many hosted clusters.
+
+Hosted cluster:: An {product-title} API endpoint that is managed by hosted control planes.
+
+Hosted control plane:: An {product-title} control plane that runs on the management cluster, which is exposed by the API endpoint of a hosted cluster. The components of a control plane include etcd, an API server, the Kubernetes controller manager, and a VPN.
+
+Hosted control plane infrastructure:: Resources on the management cluster or external cloud provider that are prerequisites to run hosted control plane processes.
+
+Management cluster infrastructure:: Network, compute, and storage resources of the management cluster.
+
+Hosted cluster infrastructure:: Network, compute, and storage resources that exist in the customer cloud account.
+
+Cluster service provider:: The user who hosts the cluster control planes and is responsible for uptime, observing the user interface for fleet-wide alerts. The cluster service provider configures the cloud account to host control planes, the user-provisioned infrastructure, which includes the host awareness of available compute, and where VMs are pulled from. This user has cluster administrator management access.
```

Comment:
```suggestion
Cluster service provider:: The role assumed by the user who hosts the cluster control planes and is responsible for uptime, observing the user interface for fleet-wide alerts. The cluster service provider persona configures the cloud account for the management cluster to host control planes, the user-provisioned infrastructure, which includes the host awareness of available compute, and where VMs are pulled from. Assuming this role provides cluster administrator access to the management cluster.
```

Reply:
I still think the sentence is too long.

Reply:
Adding the following will also help highlight and explain the scope: 

Users assuming this role are usually cluster-admins on the management cluster and have RBAC to monitor and own the availability of the HyperShift operator as well as the control planes for the tenant’s hosted clusters.